src/server/controllers/tasks.js

```diff
@@ -52,4 +52,27 @@ export class TaskManager {
       }
     });
   }
+  /**
+         *
+         *Updating a task
+         * @memberof TaskManager
+         */
+  updateTask() {
+    Task.findById(this.req.params.id).exec((error, tasks) => {
+      if (error) {
+        this.res.json({
+          error: error.message,
+        });
+      } else {
+        if (this.req.body.name && this.req.body.body) {
+          tasks.name = this.req.body.name || tasks.name;
+          tasks.body = this.req.body.body || tasks.body;
```

Comment:
What of the dates? You're still not handling the date & time, or won't those ever be updated?

Reply:
I think some of these properties will be added at further stages of development

Reply:
i am still working on just a simple functionality of creating a simple todo with just a name.I will keep on improving on it

Reply:
Im done with this...i will be handling date and time

Reply:
for they will also be updated at some time